server/tests/util/keycloak.js

```diff
@@ -21,6 +21,11 @@ const healthAuthority = {
   password: process.env.KC_TEST_HA_PWD || 'password',
 };
 
+const ministryOfHealth = {
+  username: 'test-moh',
+  password: process.env.KC_TEST_MOH_PWD || 'password',
+};
+
```

Comment:
Needed this for testing - but it is included in a different PR, and will be removed once that pr is merged.